unittests/gvisor-tests/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+
+# Careful. Globbing can't see changes to the contents of files
+# Need to do a fresh clean to see changes
+file(GLOB_RECURSE POSIX_TESTS CONFIGURE_DEPENDS ${CMAKE_SOURCE_DIR}/External/fex-gvisor-tests-bins/*_test)
+
+foreach(POSIX_TEST ${POSIX_TESTS})
+
+  string(REPLACE "/fex-gvisor-tests-bins/" ";" TEST_NAME_LIST ${POSIX_TEST})
+  list(GET TEST_NAME_LIST 1 TEST_NAME)
+  string(REPLACE "/" "-" TEST_NAME ${TEST_NAME})
+
+# Interpreter is too slow to run these tests
+#  add_test(NAME "${TEST_NAME}.int.gvisor"
```

Comment:
Remove the commented out add_test, just leave the comment as to why we are only testing JIT.

Reply:
Done